docs/part1.md

```diff
@@ -1,6 +1,6 @@
 # AWS Fargate ECS Terraform Module
```

Comment:
guess we need to update the readme here concerning all required data source and the removed DNS stuff